test/navbar.test.tsx

```diff
@@ -144,6 +144,89 @@ describe('Navbar', () => {
     expect(bootstrapNavDropdown.text()).toEqual('A1')
   })
 
+  it('should render link list item with divider when dividerAbove prop is passed as true', () => {
+    const onClickButton = sinon.spy()
+    const onChangeInput = sinon.spy()
+    const NavbarWrapper = shallow(
+      <Navbar
+        navItems={[
+          {
+            type: 'header',
+            label: 'test',
+          },
+          {
+            type: 'search',
+            placeholderText: 'Custom',
+            buttonText: 'Text',
+            buttonColor: 'secondary',
+            onClickButton,
+            onChangeInput,
+          },
```

Comment:
We can do without these because this test is all about the linked list.

Reply:
Cool, I'll take them out.  l left them in as they were included in all the other tests;  would you like me to remove all redundant elements from the navbar tests while I'm in there? :)

Reply:
That's a great idea. Yes, thank you.